### PR TITLE
Add support for engine and query params and try with snapshot version

### PR DESCRIPTION
**Pull request by aymeric-dispa**
What it does:

- Add support for engine name and account name
  - Users will now be able to specify those values
- Add support for query params
- Enable integration tests for existing databases

### CODE REVIEW

File: resources/metabase-plugin.yaml [hunk 1]
```diff
@@ -17,9 +17,19 @@ driver:
...
-        - placeholder: "engine=engine_name"
+    - name: account
+      display-name: Account name
+      required: false
+      placeholder: account name
+    - name: engine_name
+      display-name: Engine name
+      required: false
+      placeholder: engine name
+      helper-text: If not specified, the default engine will be picked.
```
Comment (ptiurin) [inline resources/metabase-plugin.yaml:28]: Do we want to add engine_url support as well?

File: src/metabase/driver/firebolt.clj [hunk 2]
```diff
@@ -46,8 +46,8 @@
       :or    {db ""}
       :as   details}]
   (let [spec {:classname "com.firebolt.FireboltDriver", :subprotocol "firebolt", :subname (str "//api.app.firebolt.io/" db), :ssl true}]
-    (-> (merge spec (select-keys details [:password :classname :subprotocol :user :subname :additional-options]))
-        (sql-jdbc.common/handle-additional-options  (select-keys details [:password :classname :subprotocol :user :subname :additional-options]))
+    (-> (merge spec (select-keys details [:password :classname :subprotocol :user :subname :account :additional-options :engine_name]))
+        (sql-jdbc.common/handle-additional-options  (select-keys details [:password :classname :subprotocol :user :subname :account :additional-options :engine_name]))
```
Comment (ptiurin) [inline src/metabase/driver/firebolt.clj]: Does order of arguments matter here?

**Review by ptiurin — APPROVED** [review]
Couple of questions, otherwise ok.

File: resources/metabase-plugin.yaml (hunk 1 above)
Comment (aymeric-dispa) [reply resources/metabase-plugin.yaml:28]: I don't think so (it is getting deprecated)

**Comment by kevinmarr** [comment]
Already created associated docs PR: https://github.com/firebolt-db/firebolt-db.github.io/pull/59

Please drop a comment in that PR once this one is merged.

**Pull request merged**